internal/metamorph/store/postgresql/postgres_test.go

```diff
@@ -52,7 +52,7 @@ func TestMain(m *testing.M) {
 		log.Fatalf("failed to create pool: %v", err)
 	}
 
-	port := "5433"
+	port := "5434"
```

Comment:
Why?

Reply:
@arkadiuszos4chain There is another test which uses the same port. There can be an error because that port is already used

Reply:
Because we had two tests using this same port, I fixed it in the same time as Kuba. His PR is already merged into main so I will revert my changes